### ISSUE TITLE
Would like to use - Update to Swift 5

### ISSUE THREAD

**Issue by realityworks** [petrmanek/Revolver#3]
Are you interested in updating to Swift 5? Would be happy to join as a contributor and promote it.

**Comment by petrmanek** [comment]
Yes, I am. I can't dedicate any capacity to such an effort at the moment, though. If you can manage, feel free fork and improve. Contributions are welcome!
